### ISSUE TITLE
client.positions() returns incomplete data

### ISSUE THREAD

**Issue by HaSiMiPa** [Voyz/ibind#6]
### to be clear, this is not a defect of iBind

`client.positions() `uses the API call `/portfolio/{accountId}/positions/{pageId}`

`client.positions()` does not always return all data fields. Instead of the expected 47 fields, it sometimes returns only 41 or 21 fields.
Sometimes means: it can be incomplete for a sequence of 10 calls, or only once in 20 calls. There is no visible pattern.
Although it is not a defect of iBind, you may be still impacted by this behaviour, depending on what data fields you plan to use.
You cannot assume all data is delivered but you need to have checks in place.

I will observe this for some time and likely create a ticket at IB. If somebody else has observed this as well or found a logic or reason, please comment here.

**Comment by Voyz** [comment]
That's true, it can happen sometimes, I've noticed it too. There's a solution I'm thinking about but not sure if I can share it here publicly. Can you drop me a quick email? You'll find it in the [README](https://github.com/Voyz/ibind) of this repo.

**Comment by HaSiMiPa** [comment]
ok, thanks. Let's see if IB provides a stability fix or a better solution in the future.
